locust/argument_parser.py

```diff
@@ -355,9 +355,16 @@ def setup_parser_arguments(parser):
         dest="web_auth",
         metavar="<username:password>",
         default=None,
-        help="DEPRECATED. See https://github.com/locustio/locust/issues/2517 Turn on Basic Auth for the web interface. Should be supplied in the following format: username:password ",
+        help="DEPRECATED: use --web-login",
```

Comment:
Change this to configargparse.SUPPRESS and let the error message speak for itself :)